### PR TITLE
fix trust region tests

### PR DESCRIPTION
**Pull request by joelberkeley**

### CODE REVIEW

File: tests/unit/acquisition/test_rule.py [hunk 1]
```diff
@@ -185,7 +185,7 @@ def test_trust_region_for_unsuccessful_local_to_global_trust_region_reduced() ->
     eps = 0.5 * (search_space.upper - search_space.lower) / 10
     previous_y_min = dataset.observations[0]
     is_global = False
-    acquisition_space = Box(dataset.query_points - eps, dataset.query_points + eps)
+    acquisition_space = Box(dataset.query_points[1] - eps, dataset.query_points[1] + eps)
```
Comment (vpicheny) [inline tests/unit/acquisition/test_rule.py]: actually it should be dataset.query_points[0]

File: tests/unit/acquisition/test_rule.py [hunk 2]
```diff
@@ -162,7 +162,7 @@ def test_trust_region_for_successful_local_to_global_trust_region_increased() ->
...
```
Comment (vpicheny) [inline tests/unit/acquisition/test_rule.py]: my bad it should be dataset.query_points[0]

Comment (joelberkeley) [reply tests/unit/acquisition/test_rule.py]: tests don't seem to care :s we definitely need more acquisition rule tests

File: tests/unit/acquisition/test_rule.py (hunk 1 above)
Comment (joelberkeley) [reply tests/unit/acquisition/test_rule.py]: are you certain?

**Review by vpicheny — APPROVED** [review]

File: tests/unit/acquisition/test_rule.py (hunk 2 above)
Comment (joelberkeley) [reply tests/unit/acquisition/test_rule.py]: it's cos the acquisition space is only used in the algorithm to work out its size
```
            tf.reduce_prod(
                state.acquisition_space.upper - state.acquisition_space.lower
            )
```
and we don't test the `query_point` returned

**Pull request merged**